lib/src/combined_wrappers/combined_map.dart

```diff
@@ -39,14 +41,56 @@ class CombinedMapView<K, V> extends UnmodifiableMapBase<K, V> {
 
   /// The keys of [this].
   ///
-  /// The returned iterable has efficient `length` and `contains` operations,
-  /// based on [length] and [containsKey] of the individual maps.
+  /// The returned iterable has efficient `contains` operations, based on
+  /// [containsKey] of the individual maps.
```

Comment:
This comment isn't quite accurate. We optimize based on `keys.contains` of the individual maps, we _expect_ those are in turn optimized based on `containsKey`, but we can't know for sure. Is it worth making a distinction?

Reply:
I had a similar thought.... but I didn't feel like it was worth changing the existing comment?

Reply:
I went ahead and updated it